packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -37,6 +37,9 @@ export class ChoiceList extends React.PureComponent {
       choiceProps.onChange = this.props.onChange;
       choiceProps.size = this.props.size;
       choiceProps.type = this.props.type;
+      choiceProps.className = classNames(choiceProps.className, {
```

Comment:
I think this should be applied to the `inputClassName` not `className`. This is consistent with how `ds-c-field--error` works as well, and should simplify the CSS

Reply:
Hmmm, with choice items we hide the input element and are styling the before element based on the label `.ds-c-choice:checked + label::before`?

Reply:
Right, that part doesnt change. What I'm saying is that we should be applying the `ds-c-choice--error` class to the `input`, similar to the other modifiers. 
<img width="559" alt="Screen Shot 2020-09-23 at 2 07 50 PM" src="https://user-images.githubusercontent.com/5424713/94058095-38f1f180-fda6-11ea-861b-9b9d716cf715.png">